File: Go/Math/get_biggest_number_in_slice.go
package main

import "fmt"

func main() {
	// Create a slice of random data.
	randomNumbers := []int{0, 1, 2, 3, 4}
	// Get the biggest number in a slice.
	fmt.Println(getBiggestNumberInSlice(randomNumbers))
}

// Get the biggest number in a given slice.
func getBiggestNumberInSlice(slice []int) int {
	var biggestNumber int
	for _, value := range slice {
		if value > biggestNumber {
			biggestNumber = value
		}
	}
	return biggestNumber
}

File: Dart/Math/get_biggest_number_in_slice.dart
import "dart:core";
import "dart:math";

void main() {
  // Create of list of numbers.
  final List<int> randomSlice = <int>[1, 2, 3];
  // Find the biggest number in the given slice.
  print(getBiggestNumberInSlice(randomSlice));
}

// Get the biggest number in a given slice.
int getBiggestNumberInSlice(final List<int> givenSlice) {
  return givenSlice.reduce(max);
}
